028-service-binding.md

```diff
@@ -55,18 +55,23 @@ Required changes:
 - Update the secret created for the `KafkaUser` to include the username when the `KafkaUser` is using type SASL SCRAM
 - Update the `Kafka` CR to include a `status.binding` field, where the `name` refers to a new secret containing both the cluster certificate and bootstrapServers
 - Add a new secret that contains the bootstrap servers for each listener and the cluster certificate information
-  - e.g. for listeners the secret data could be formatted like:
+  - e.g. for listeners, the secret data could be formatted to have one string with all listeners:
     ```yaml
     data:
-      listeners:
-        - name: internal
-          bootstrapServers: kafka.svc:9092
-        - name: external
-          bootstrapServers: myhost.com:443
+      listeners: aW50ZXJuYWxfa2Fma2Euc3ZjOjkwOTIsZXh0ZXJuYWxfbXlob3N0LmNvbTo0NDM= # when base64 decoded something like internal_kafka.svc:9092,external_myhost.com:443
```

Comment:
why not using just json, instead of encoding it?

Reply:
From the Kubernetes documentation "the values for all keys in the data field have to be base64-encoded strings.", but we could use a json string and base64 encode that. I think if we did go for this approach we would have to think carefully about the best format to use.

Reply:
> From the Kubernetes documentation "the values for all keys in the data field have to be base64-encoded strings."

The values are encoded automatically at secret creation and decoded automatically at secret projection/mounting. Hence, for communication/documentation purpose, one can define keys under `stringData`, similar to https://kubernetes.io/docs/concepts/configuration/secret/#bootstrap-token-secrets